operator/v1/types_ingress.go

```diff
@@ -536,6 +536,43 @@ type HostNetworkStrategy struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	Protocol IngressControllerProtocol `json:"protocol,omitempty"`
+
+	// HTTPPort is the port on the host which should be used to listen for
+	// HTTP requests. This field should be set when port 80 is already in use.
```

Comment:
For each new field, please add a sentence describing the default value.  The default value that is specified by the kubebuilder tag doesn't show in generated Swagger or the OpenAPI schema.